spec/controllers/custom_mail_controller_spec.rb

```diff
@@ -68,6 +68,14 @@
         expect(response.body).to eq("The mails were queued for sending!")
       end
     end
+
+    describe "#index" do
+      it "shows the list of custom views" do
+        get custom_mail_path(@mail)
```

Comment:
You'll have to do `get custom_mails_path` to actually check the index route, this should check for the index route. 
This is also the reason why the test coverage is failing.

Reply:
Thanks, should be fixed now